test/commands/dev/index.test.ts

```diff
@@ -947,6 +976,59 @@ describe('local dev environment', function () {
       const hello_world_service = runCompose.firstCall.args[0].services[hello_api_ref] as any;
       expect(hello_world_service.environment.compose_escaped_variable).to.equal('variable_split_$$_with_dollar$$signs');
     })
+  
+  test
```

Comment:
You need a few more tests.
1. What happens if they give a bad env name?
2. Make sure we ask for an account if the account is not given.